projects/container_bench/testing/remote_access.py

```
@@ -0,0 +1,162 @@
+    print(extra_vars_yaml_content)
+    print(extra_vars_yaml_content, file=extra_vars_file)
+    extra_vars_file.flush()
+
+    #
+
+    return base_work_dir
+
+
+def run_with_ansible_ssh_conf(
+        base_work_dir, cmd,
+        extra_env={},
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)